src/utils.test.js

```diff
@@ -190,6 +191,21 @@ describe('utils', () => {
     process.env = { ...oldEnv };
   });
 
+  test('getEventEntitySize', () => {
+    expect(utils.getEventEntitySize()).toBe(MAX_ENTITY_SIZE);
+    const oldEnv = Object.assign({}, process.env);
+    process.env = { ...oldEnv, MAX_EVENT_ENTITY_SIZE: ' 2048' };
+    expect(utils.getEventEntitySize()).toBe(2048);
+    process.env = { ...oldEnv };
+  });
+
+  test('getEventEntitySize NaN', () => {
+    const oldEnv = Object.assign({}, process.env);
+    process.env = { ...oldEnv, MAX_EVENT_ENTITY_SIZE: 'A 2048' };
+    expect(utils.getEventEntitySize()).toBe(MAX_ENTITY_SIZE);
+    process.env = { ...oldEnv };
+  });
+
```

Comment:
can you also check the event+prune itself?
I.e.  give it a long event and check that it is pruned..

Reply:
Not following you, what's wrong with the prune tests?

Reply:
you only checked the function `getEventEntitySize`, and not the actual change in `getFunctionSpan`